cmd/krel/cmd/release_notes.go

```diff
@@ -85,6 +89,30 @@ func init() {
 		"version tag for the notes",
 	)
 
+	releaseNotesCmd.PersistentFlags().StringVarP(
+		&releaseNotesOpts.sigReleaseFork,
+		"sigrelease-fork",
```

Comment:
Same as above; use two options with the flags `draft-org`, `draft-repo`

Reply:
Done. New flags are in, old ones gone.